packages/frontend/src/hooks/organisation/useOrganisation.ts

```diff
@@ -13,4 +13,6 @@ export const useOrganisation = () =>
     useQuery<Organisation, ApiError>({
         queryKey: ['organisation'],
         queryFn: getOrganisation,
+        retry: false,
+        refetchOnMount: false,
```

Comment:
So I'm not sure whether we want this. Currently the way this query works is that whenever a component renders, it checks the organisation API to make sure it has the most up to date value.

Now, when components mount, it won't actually fetch the latest version from the API but will rely on what we have in memory. Is that what we want?

the alternative is to find out what component depends on this `useOrganization` hook and why it is rendering so many times.

https://react-query-v3.tanstack.com/guides/important-defaults

Reply:
Yeah, I think it is happening here https://github.com/lightdash/lightdash/blob/main/packages/frontend/src/components/ChartConfigPanel/ChartConfigTabs.tsx#L160

Reply:
Putting it in the provider should be ok because the provider only renders once. We just need to be aware that the components lower down the stack won't get the latest `/org` data until the provider refreshes

Reply:
I actually don't mind the many calls to `/org`, I doubt it affects performance. It's more that the refetch triggers the chart to flicker. I don't want to pivot this issue into trying to stop all the `/org` fetches. By fetching constantly we know we always have the most recent data when each component loads, which is much simpler than trying to manage a cache of the state around the app

Reply:
I think so yes, I don't get why that trigger happens

Reply:
Some component in the chart config must be using the `useOrganization` hook instead of using the data from the `useApp` hook. 🤔